README.md

```diff
@@ -24,9 +24,25 @@ Note that the order of operations is important. The following will result in an
 
 ## Installation
 
-First, ensure that you have Go Version 1.22 on you system. Clone this repository and run 
+### Build from Source
 
-`go build -o festerize main.go`
+First, ensure that you have Go Version 1.23 on you system. Clone this repository and run 
```

Comment:
I wonder if we should just say "latest" or say "...you have at least Go Version 1.23," since I don't want to keep updating the README when we update Go versions.

Reply:
I think latest will make the most sense